WcaOnRails/app/jobs/generate_chore.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class GenerateChore < SingletonApplicationJob
+  queue_as :default
+
+  def perform
+    # Randomly select a member who has been there for at least a month
+    members = Team.wst.current_members.select { |m| m.start_date < 1.month.ago }
+    chosen = rand(0..(members.size-1))
```

Comment:
I believe you can use `members.sample` to get a random element =)

Reply:
Ah nice, I'll update that!